src/Query/Builder.php

```diff
@@ -1534,6 +1537,24 @@ public function options(array $options)
         return $this;
     }
 
+    /**
+     * Set the read preference for the query
+     *
+     * @see https://www.php.net/manual/en/class.mongodb-driver-readpreference.php
+     *
+     * @param  string $mode
+     * @param  array  $tagSets
+     * @param  array  $options
+     *
+     * @return $this
+     */
+    public function readPreference(string $mode, ?array $tagSets = null, ?array $options = null): static
```

Comment:
Just want to confirm that you don't need the ability to _clear_ a ReadPreference previously assigned to the builder.

Reply:
Most of the operations using the query builder cannot be unset. 
For this config, it's still possible to overwrite with an other read preference.